pkg/mutation/assignmeta_mutator.go

```diff
@@ -0,0 +1,125 @@
+package mutation
+
+import (
+	"fmt"
+	"reflect"
+
+	"github.com/google/go-cmp/cmp"
+	mutationsv1alpha1 "github.com/open-policy-agent/gatekeeper/apis/mutations/v1alpha1"
+	"github.com/open-policy-agent/gatekeeper/pkg/mutation/path/parser"
+	"github.com/pkg/errors"
+	corev1 "k8s.io/api/core/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/apis/meta/v1/unstructured"
+)
+
+var (
+	labelsValidSubPath = []parser.Node{
+		&parser.Object{
+			Reference: "metadata",
+		},
+		&parser.Object{
+			Reference: "labels",
+		},
+	}
+
+	annotationValidiSubPath = []parser.Node{
```

Comment:
nit: is `annotationValidiSubPath` intentional or a typo?

Reply:
Was a typo, fixed it